src/storage/aggregator_test.cc

```diff
@@ -84,18 +84,18 @@ bool UpdateAggr(std::shared_ptr<Aggregator> aggr, codec::RowBuilder* row_builder
         uint32_t row_size = row_builder->CalTotalLength(6 + str.size());
         encoded_row.resize(row_size);
         row_builder->SetBuffer(reinterpret_cast<int8_t*>(&(encoded_row[0])), row_size);
-        row_builder->AppendString("id1", 3);
-        row_builder->AppendString("id2", 3);
-        row_builder->AppendTimestamp(static_cast<int64_t>(i) * window_size / 2);
-        row_builder->AppendInt32(i);
-        row_builder->AppendInt16(i);
-        row_builder->AppendInt64(i);
-        row_builder->AppendFloat(static_cast<float>(i));
-        row_builder->AppendDouble(static_cast<double>(i));
-        row_builder->AppendDate(i);
-        row_builder->AppendString(str.c_str(), str.size());
-        row_builder->AppendNULL();
-        row_builder->AppendInt32(i % 2);
+        (void)row_builder->AppendString("id1", 3);
```

Comment:
good first issue for those ?

Reply:
all warnings about these function have been fixed